twake/backend/node/src/core/platform/framework/pubsub.ts

```diff
@@ -0,0 +1,36 @@
+import { EventEmitter } from "events";
+import { TrackerEventActions } from "../../../core/platform/services/tracker/types";
```

Comment:
This is not related to tracker, this event bus is independant. The type should also be local to the pubsub.ts file (put it in this file)